modules/navier_stokes/test/tests/finite_volume/wcns/natural_convection/natural_circulation_pipe.i

```diff
@@ -115,7 +119,7 @@ gamma = 1.4
   [mu_rampdown_fn]
     type = PiecewiseLinear
     x = '0    0.5  1   5  10 100 1000 2000'
-    y = '1000 1000 100 10 1  1   1    0'
+    y = '1000 1000 100 10 1  1   1    1'
```

Comment:
Great catch!

Reply:
Since i need to divide the Darcy friction factor with mu to make it consistent with the previous kernel and don't change the gold file, i can't have mu = 0.0. That changes the results a little bit so i had to increase the relative error to 1e-4, which i think is sufficient.